src/dbnode/storage/index.go

```diff
@@ -1136,14 +1189,25 @@ func (i *nsIndex) query(
 }
 
 func (i *nsIndex) execBlockQueryFn(
+	ctx context.Context,
 	cancellable *resource.CancellableLifetime,
 	block index.Block,
 	query index.Query,
 	opts index.QueryOptions,
 	state *asyncQueryExecState,
 	results index.BaseResults,
+	logFields []opentracinglog.Field,
 ) {
-	blockExhaustive, err := block.Query(cancellable, query, opts, results)
+	logFields = append(logFields, []opentracinglog.Field{
```

Comment:
Don’t need to create the second slice when appending multiple values. ie can do this:

```
logFields = append(logFields, xopentracing.Time(...), xopentracing.Time(...))
```